app/controllers/admin/works_controller.rb

```diff
@@ -3,12 +3,14 @@
 # Started with generated code from Rails 5.2 scaffold.
 #
 # We'll probably handle `show` in a different controller, for now no show.
+require 'mimemagic'
```

Comment:
Even if we are using mimemagic... I'm surprised we need to explicitly require it. I think the correct "fix" might be adding mimemagic to our Gemfile (if we are using it directly, we should express it as a dependency directly) -- and then I think bundler would take care of it requiring it on boot and we wouldn't need to. 

If we stop using it in this file entirely, that would also eliminate the need for the "require".

Reply:
Got rid of it!